### PR TITLE
add stylelint config info

### PR DESCRIPTION
**Pull request by hanakin**

### CODE REVIEW

File: development/development/phpstorm.rst [hunk 1]
```diff
@@ -72,6 +72,8 @@ The default inspection settings should work just fine. However there are a coupl
 * phpBB uses fully qualified namespaces, so you can turn off this inspection warning **PHP > Code Style > Unnecessary fully qualified name**.
 * You may enable additional JavaScript inspections. Under **JavaScript > Code quality tools**, you can enable JSCS, JSHint, JSlint and/or ESLint. phpBB comes with configuration files for JSHint and JSCS. With these inspections enabled, you can use the configuration files provided by phpBB instead of PhpStorm's default settings. To do so, point PhpStorm to the custom configuration files under **Language & Frameworks > JavaScript > Code Quality Tools**. Point JSHint to `.jshintrc` and point JSCS to `.jscsrc`.
 
+.. warning:: Stylelint is not bundled with PhpStorm. The Stylelint tool is run through NodeJS. To use Stylelint, make sure the NodeJS framework is downloaded and installed on your computer. Then you can follow these `Instructions <https://www.jetbrains.com/help/phpstorm/stylelint.html>`_.
+
```
Comment (VSEphpbb) [inline development/development/phpstorm.rst]: @hanakin doesn't work for me. I had a very difficult time getting stylelint to work for me in phpStorm due to the stylelint-order component.

I always get this error on CSS pages whenever Stylint tries to inspect:
> Error: Could not find "stylelint-order". Do you need a `configBasedir`?

Somehow I got it working on one project by installing stylelint-order and/or stylelint-declaration-block-order. Then I had to change .stylelintrc to swap `order/declaration-block-order` for `order/order` and `order/declaration-block-properties-specified-order` for `order/properties-order`. Apparently these declaration-properties are deprecated. But trying to get this to work again on a new project and failing.

Comment (VSEphpbb) [reply development/development/phpstorm.rst]: Update: OK, weird but this time I got it to finally work, just by installing stylelint-order through PhpStorm (I had also installed stylelint-declaration-block-order previously so I can't say if that has anything to do with it working). But I think it's using an old stylelint-order (o.3.0) while I believe the current version has changed the names as I mentioned above -> https://github.com/hudochenkov/stylelint-order/blob/master/CHANGELOG.md#040

Comment (hanakin) [reply development/development/phpstorm.rst]: Yes it used to be part of stylelint then they made it a plugin. I do not use phpstorm as atom and studio web are the Main stream front-end editors. so not sure about any of this... but u should be fine installing stylelint and any of the plugins that we are using. I'll update it with what those when I get a chance. It should just be stylelint-order for now I believe

**Comment by DavidIQ** [comment]
Since this is just providing a general overview of what's needed and does not look to be a comprehensive guide on the topic, I'll just merge for now.

**Pull request merged**